src/NuGet.Clients/NuGet.VisualStudio.OnlineEnvironment.Client/NuGetClientPackage.vsct

```diff
@@ -36,7 +36,7 @@
          group as the part of a menu contained between two lines. The parent of a group
          must be a menu. -->
     <Groups>
-      <Group guid="guidNuGetClientPackageCmdSet" id="NuGetClientMenuGroup" priority="0x0600">
+      <Group guid="guidNuGetClientPackageCmdSet" id="nuGetClientMenuGroup" priority="0x0600">
```

Comment:
Tools VSCT has `guidDialogCmdSet`, and yours is named slightly differently, but not in a way that helps me understand it's specific to OE (`guidNuGetClientPackageCmdSet`). Is this naming important the way it is? Can we rename it now without much to change?

Parity with Tools should probably be a goal. From my [PR](https://github.com/NuGet/NuGet.Client/pull/3346/files), I'm using  `<Group guid="guidPackageManagementCmdSet" id="idPackageManagementGroup" priority="0x0100">`

Reply:
The names are not super important, the guids are the only thing that matters.
These context menus are a lot more minimal.

I think they can be refactored to achieve some parity, but honestly I'd do the reverse. 
`guidDialogCmdSet` is confusing, too generic.

If you want I can create an issue for that parity, but right now, I'd prefer to keep this change more minimal if possible.

Reply:
Yes, please create an issue to follow-up here.

Reply:
Created an issue for follow up: https://github.com/NuGet/Home/issues/9472